doc/api/wasi.md

```diff
@@ -16,21 +16,18 @@ import { WASI } from 'wasi';
 import { argv, env } from 'node:process';
 
 const wasi = new WASI({
+  version: 'wasi_snapshot_preview1',
   args: argv,
   env,
   preopens: {
     '/sandbox': '/some/real/path/that/wasm/can/access',
   },
 });
 
-// Some WASI binaries require:
-//   const importObject = { wasi_unstable: wasi.wasiImport };
-const importObject = { wasi_snapshot_preview1: wasi.wasiImport };
-
```

Comment:
We appear to be losing this information, which could be important to some users. I think we either need to keep this somewhere in the docs, or support passing in "wasi_unstable" as a version which would return the same API as wasi_snapshot_preview1, but with a different version string.

Reply:
I'm happy to either add it back to the doc or add to "wasi_unstable" to what  can be passed in as a version depending on what people think is best.

Reply:
It seems like a pretty easy thing to support since it would be an alias for preview1, so I'd suggest supporting it and removing this note from the docs (but we would need to document that "wasi_unstable" is a valid version).

Reply:
will do